toolchain/check/import.cpp

```diff
@@ -500,12 +497,14 @@ static auto AddNamespaceFromOtherPackage(Context& context,
     -> SemIR::InstId {
   auto namespace_type_id =
       context.GetBuiltinType(SemIR::BuiltinInstKind::NamespaceType);
-  auto [new_scope_id, namespace_inst_id] = CopySingleNameScopeFromImportIR(
-      context, namespace_type_id, /*copied_namespaces=*/nullptr, import_ir_id,
-      import_inst_id, import_ns.name_scope_id, parent_scope_id, name_id);
-  context.name_scopes()
-      .Get(new_scope_id)
-      .import_ir_scopes.push_back({import_ir_id, import_ns.name_scope_id});
+  auto [new_scope_id, namespace_inst_id, is_duplicate] =
+      CopySingleNameScopeFromImportIR(
+          context, namespace_type_id, /*copied_namespaces=*/nullptr,
+          import_ir_id, import_inst_id, import_ns.name_scope_id,
+          parent_scope_id, name_id);
+  auto& scope = context.name_scopes().Get(new_scope_id);
+  scope.is_closed_import = !is_duplicate;
```

Comment:
I found this a bit confusing; I wonder if better naming for `is_duplicate` would help. Something like `is_duplicate_of_namespace_in_current_package` would be clearer as to why it matters here, but maybe there's a shorter way of expressing the same thing? (`found_in_current_package`?)

Reply:
Went with the verbose name - it's only written 3 times (now that there's a named type, which reduces some of the mentions) so being especially long doesn't seem too bad

Reply:
Works for me, thanks.